src/pyodide-py/pyodide/_base.py

```diff
@@ -42,6 +43,31 @@ def open_url(url: str) -> StringIO:
     return StringIO(req.response)
 
 
+def quiet_code(code: str) -> bool:
+    """
+    Does the last nonwhitespace character of code is a semicolon?
+
+    This can be overridden to customize the way eval_code is silenced.
+    """
```

Comment:
Could you add a docstring example here,

```
Examples
--------
>>> quiet_code('x + 1')
False
>>> quiet_code('x +1 ;')
True
>>> quiet_code('x +1 # comment ;')
False
```
which would help understanding what it does also serve as a doctest